plugins/com.google.cloud.tools.eclipse.ui.util/META-INF/MANIFEST.MF

```diff
@@ -32,6 +32,7 @@ Import-Package: com.google.cloud.tools.eclipse.ui.util,
  org.eclipse.ui.browser;ui.workbench=split,
  org.eclipse.ui.console,
  org.eclipse.ui.handlers,
+ org.eclipse.ui.ide,
```

Comment:
Tried that. Turned out this is used for other reasons in other classes not in this PR.